pandas/tests/extension/decimal/test_decimal.py

```diff
@@ -92,15 +92,21 @@ def assert_frame_equal(self, left, right, *args, **kwargs):
 
 
 class TestDtype(BaseDecimal, base.BaseDtypeTests):
-    pass
+
+    def test_array_type_with_arg(self, data, dtype):
+        assert dtype.construct_array_type('foo') is DecimalArray
 
 
 class TestInterface(BaseDecimal, base.BaseInterfaceTests):
     pass
 
 
 class TestConstructors(BaseDecimal, base.BaseConstructorsTests):
-    pass
+
+    @pytest.mark.xfail(reason="not implemented constructor from dtype")
+    def test_from_dtype(self, data):
+        # construct from our dtype & string dtype
+        pass
```

Comment:
Can you actually implement this? (only change needed would be to register DecimalDtype I think?) 
As that way we actually have a test for external dtypes registering?